Pipfile

```diff
@@ -27,6 +27,7 @@ doublex-expects = "*"
 pep8 = "*"
 autopep8 = "*"
 rope = "*"
+PyHamcrest = "*"
```

Comment:
These are the matchers used to match different Mock calls. So we don't have to use <ANY_ARGS> everywhere.